Assets/Dojo/Runtime/WorldManager.cs

```diff
@@ -11,18 +11,33 @@ namespace Dojo
     public class WorldManager : MonoBehaviour
     {
         [Header("RPC")]
```

Comment:
Get rid of the Header? As they are now hidden from inspector

Reply:
Yes, I forgot to delete it!